pkg/analyze/host_certificate.go

```diff
@@ -23,52 +23,34 @@ func (a *AnalyzeHostCertificate) IsExcluded() (bool, error) {
 func (a *AnalyzeHostCertificate) Analyze(
 	getCollectedFileContents func(string) ([]byte, error), findFiles getChildCollectedFileContents,
 ) ([]*AnalyzeResult, error) {
-	hostAnalyzer := a.hostAnalyzer
 
-	collectorName := hostAnalyzer.CollectorName
+	collectorName := a.hostAnalyzer.CollectorName
 	if collectorName == "" {
 		collectorName = "certificate"
 	}
-	name := filepath.Join("host-collectors/certificate", collectorName+".json")
-	contents, err := getCollectedFileContents(name)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to get collected file")
-	}
-	status := string(contents)
-
-	trimmedStatus := strings.TrimSpace(status)
-
-	var coll resultCollector
-
-	for _, outcome := range hostAnalyzer.Outcomes {
-		result := &AnalyzeResult{Title: a.Title()}
-
-		if outcome.Fail != nil {
-			if outcome.Fail.When == "" || outcome.Fail.When == trimmedStatus {
-				result.IsFail = true
-				result.Message = outcome.Fail.Message
-				result.URI = outcome.Fail.URI
 
-				coll.push(result)
-			}
-		} else if outcome.Warn != nil {
-			if outcome.Warn.When == "" || outcome.Warn.When == trimmedStatus {
-				result.IsWarn = true
-				result.Message = outcome.Warn.Message
-				result.URI = outcome.Warn.URI
+	const nodeBaseDir = "host-collectors/certificate"
```

Comment:
wondering if we should make this more consistent with some of the other collectors and always reference a variable from the collector code since that will always be source of truth?

Reply:
ack, I'll do this in a follow PR